JWT.xcodeproj/project.xcworkspace/xcshareddata/JWT.xcscmblueprint

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This is a file that Xcode uses to track `git` state and it local user data. Thus it shouldn't be committed to the repository.